django_celery_beat/templates/admin/djcelery/change_periodictask_form.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'admin/change_form.html' %}
+
+{% block admin_change_form_document_ready %}
+    {{ block.super }}
+
+    {{ readable_crontabs|json_script:"readable-crontabs" }}
+    <script id="periodic-task-overrides">
+        const readableCrontabs = JSON.parse(django.jQuery("#readable-crontabs").text());
+        var originalCrontabHelp = "{{ adminform.form.fields.crontab.help_text }}";
+
+        var updateCrontabHelp = function(additional) {
+            django.jQuery(".field-crontab .help").html(
+                `${originalCrontabHelp}<br/>Translation: ${additional}`);
+        };
+
+        django.jQuery(".field-crontab_translation").hide()
```

Comment:
is this django.jQuery being used from django admin itself? is it possible to switch to plain vanilla JS here? ES 6 without any dependencies?

Reply:
From [django documentation](https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#jquery):
> To avoid conflicts with user-supplied scripts or libraries, Django’s jQuery (version 3.6.0) is namespaced as django.jQuery. 

Since this namespaced jQuery comes with Django admin, this seems like the best choice for using jQuery. It's probably possible to use ES 6 w/o dependencies but jQuery is not an extra dependency, so I think using this is ok.